app/assets/javascripts/components/activity/activity_table_row.jsx

```diff
@@ -65,7 +65,9 @@ const ActivityTableRow = React.createClass({
 
     let diffViewer;
     if (this.props.revision && this.props.revision.api_url) {
-      diffViewer = <DiffViewer revision={this.props.revision} />;
+      diffViewer = <DiffViewer revision={this.props.revision}
```

Comment:
Ah, yes. In the cumulative diff usage (when it is provided both a revision and a first revision), the revision does not include the character count. In this context, it is showing all the changes to an article over a large span of time, so we don't want to show the character count of that last revision. If we were to show anything, it would be the cumulative change in characters across the diff range displayed, which we'd need to get from the Wikipedia API rather than passing it in as props.

In any case, we should be able to do this without adding a new required prop to the component.